lib/init.js

```diff
@@ -2,15 +2,31 @@
 
 module.exports = init
 
+var path = require('path')
 var log = require('npmlog')
 var npm = require('./npm.js')
+var npx = require('libnpx')
 var initJson = require('init-package-json')
 var output = require('./utils/output.js')
 var noProgressTillDone = require('./utils/no-progress-while-running').tillDone
 
 init.usage = 'npm init [--force|-f|--yes|-y]'
```

Comment:
This should be updated. You should document this feature in `doc/cli/npm-init.md` as well.